config/environments/production.rb

```diff
@@ -76,7 +69,17 @@
   config.active_support.deprecation = :notify
 
   # Use default logging formatter so that PID and timestamp are not suppressed.
-  # config.log_formatter = ::Logger::Formatter.new
+  config.log_formatter = ::Logger::Formatter.new
```

Comment:
The Publishing API's Rails upgrade diff leaves this commented out. I've just done the same.